src/main/java/com/epam/lowcost/util/EndPoints.java

```diff
@@ -0,0 +1,11 @@
+package com.epam.lowcost.util;
+
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class EndPoints {
+    public static final String REDIRECT_TICKETS_SELF = "redirect:/tickets/self";
```

Comment:
no no no. I said about small parts. `/tickets` , `/self`, `/user`. You should create basis and then use it how small parts

Reply:
or without `/` ... Check useful cases in google

Reply:
Extracted endpoints to utility class.